apiserver/cache/redis.go

```diff
@@ -106,3 +108,32 @@ func (r *Cache) SaveJti(ctx goContext.Context, userId int, jti string, expiratio
 func (r *Cache) FlushAll(ctx goContext.Context) error {
 	return r.client.FlushAll(ctx).Err()
 }
+
+func testInvoiceRateLimiterKey(userId int) string {
+	return fmt.Sprintf("test:invoices:userId:%d", userId)
+}
+
+func (r *Cache) IncrementTestInvoiceCounter(ctx goContext.Context, userId int) (int, error) {
```

Comment:
let's add comment above function definition. So nobody has to study code to see how exactly this limiting works.
Explain that expiration window is tracked from 1st test invoice created and when window is over, new one is created, so even limit is `20` in `24h` one can create `39` test invoices in a single minute

Longer comment is better than unclear one (maybe example(s) would be nice)